libs/core/src/components/pds-radio/stories/pds-radio.docs.mdx

```diff
@@ -16,9 +16,9 @@ Radio components provide users a way to select only one option from a list of tw
 ### Default
 
 <Canvas mdxSource={`
-  <pds-radio component-id="default" label="Label" />
+  <pds-radio component-id="default0" label="Label" />
 `}>
-  <pds-radio component-id="default" label="Label" />
+  <pds-radio component-id="default0" label="Label" />
```

Comment:
This should have been a separate commit but it resolves a documentation issue where clicking the label would not check the radio.